frontend/src/components/BreadCrumb.tsx

```diff
@@ -0,0 +1,66 @@
+'use client'
+
+import { faChevronRight } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { Breadcrumbs, BreadcrumbItem } from '@heroui/react'
+import Link from 'next/link'
+import { usePathname } from 'next/navigation'
+
+const capitalize = (str: string) => str.charAt(0).toUpperCase() + str.slice(1).replace(/-/g, ' ')
```

Comment:
We may have capitalize function in utils. Please use or extend that [one](https://github.com/OWASP/Nest/blob/main/frontend/src/utils/capitalize.ts).